travis/tests/INFO.md

```diff
@@ -1 +1,30 @@
-The directory `travis/tests/` contains the behaviour tests.
+The directory `travis/tests/` contains the **behaviour tests** for
+AtChem2. These tests check the overall behaviour of the model for a
+range of possibile configurations.
+
+- `firstorder`, `secondorder`: generic chemical mechanism (1 reaction, 2 species).
+
+- `static`: generic chemical mechanism (2 reactions, 2 species).
```

Comment:
You may wish to document that the main purpose of `firstorder`, `secondorder` and `static` are as "manufactured solutions" - we know the exact solutions to the ODEs so they act as validation checks.